deploy/edge_stage_frontend_testing.yml

```diff
@@ -0,0 +1,183 @@
+            value: ${IQE_FILTER_EXPRESSION}
+          - name: IQE_LOG_LEVEL
+            value: ${IQE_LOG_LEVEL}
+          - name: IQE_REQUIREMENTS
+            value: ${IQE_REQUIREMENTS}
+          - name: IQE_PARALLEL_ENABLED
+            value: ${IQE_PARALLEL_ENABLED}
+          - name: IQE_REQUIREMENTS_PRIORITY
+            value: ${IQE_REQUIREMENTS_PRIORITY}
+          - name: IQE_TEST_IMPORTANCE
+            value: ${IQE_TEST_IMPORTANCE}
+          - name: DYNACONF_IQE_VAULT_LOADER_ENABLED
+            value: "true"
+          - name: DYNACONF_IQE_VAULT_VERIFY
+            value: "true"
+          - name: DYNACONF_IQE_VAULT_URL
+            valueFrom:
+              secretKeyRef:
+                key: url
+                name: iqe-vault
+                optional: true
+          - name: DYNACONF_IQE_VAULT_MOUNT_POINT
+            valueFrom:
+              secretKeyRef:
+                key: mountPoint
+                name: iqe-vault
+                optional: true
+          - name: DYNACONF_IQE_VAULT_ROLE_ID
+            valueFrom:
+              secretKeyRef:
+                key: roleId
+                name: iqe-vault
+                optional: true
+          - name: DYNACONF_IQE_VAULT_SECRET_ID
+            valueFrom:
+              secretKeyRef:
+                key: secretId
+                name: iqe-vault
+                optional: true
+          resources:
+            limits:
+              cpu: ${IQE_CPU_LIMIT}
+              memory: ${IQE_MEMORY_LIMIT}
+            requests:
+              cpu: ${IQE_CPU_REQUEST}
+              memory: ${IQE_MEMORY_REQUEST}
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+          volumeMounts:
+            - name: sel-downloads
+              mountPath: /sel-downloads
+        - name: edge-sel-${TEST_TYPE}-tests-${IMAGE_TAG}-${UID}
+          image: ${IQE_SEL_IMAGE}
+          env:
+            - name: _JAVA_OPTIONS
+              value: ${SELENIUM_JAVA_OPTS}
+            - name: VNC_GEOMETRY
+              value: ${VNC_GEOMETRY}
+            - name: SE_NODE_SESSION_TIMEOUT
+              value: ${SE_NODE_SESSION_TIMEOUT}
+          resources:
+            limits:
+              cpu: ${SELENIUM_CPU_LIMIT}
+              memory: ${SELENIUM_MEMORY_LIMIT}
+            requests:
+              cpu: ${SELENIUM_CPU_REQUEST}
+              memory: ${SELENIUM_MEMORY_REQUEST}
+          volumeMounts:
+            - name: sel-shm
+              mountPath: /dev/shm
+            - name: sel-downloads
+              mountPath: /home/selenium/Downloads
+parameters:
+- name: IMAGE_TAG
+  value: ''
+  required: true
+- name: UID
+  description: "Unique job name suffix"
+  generate: expression
+  from: "[a-z0-9]{6}"
+- name: IQE_IMAGE
+  description: "container image path for the iqe plugin"
+  value: quay.io/cloudservices/iqe-tests:edge
+- name: ENV_FOR_DYNACONF
+  value: stage_proxy
+- name: USE_BETA
+  value: "true"
```

Comment:
Will there be new features coming in to edge?

Reply:
Nothing planned as of now

Reply:
beta tends to be a little unstable with the amount of other service changes impacting your tests. If there is nothing new planned, maybe don't use beta?

Reply:
We can overide this in the app-sre as well right or should I set it false here

Reply:
I have switched the flag to false